elasticsearch/elasticsearch_test.go

```diff
@@ -260,8 +260,8 @@ func (s *Suite) TestPutMapping(c *gc.C) {
 		"testtype": map[string]interface{}{
 			"properties": map[string]interface{}{
 				"foo": map[string]interface{}{
-					"stored": true,
-					"type":   "string",
+					//"stored": true,
```

Comment:
Either delete it or leave it, please. Commenting out isn't great. (alternatively leave a comment stating why the line is commented out)